### ISSUE TITLE
No autocomplete when accessing method/property on new line

### ISSUE THREAD

**Issue by deevus** [pbogut/deoplete-padawan#26]
Example:

```php
$foo = new Foo;

// does autocomplete
$qux = $foo->bar()->baz();

// doesnt autocomplete
$qux = $foo
    ->bar()
    ->baz();
```

I'm not sure whether the issue in in `deoplete-padawan`, but I would like it to work. Any ideas?

**Comment by mkusher** [comment]
looks like the issue is in padawan.php, but not sure

**Comment by pbogut** [comment]
Yep, looks like padawan.php. Plugin itself has issue as well because its not sending query right after `->` when typed in new line, but when at least one letter is provided. But padawan.php is not returning completion even when query is sent.

**Comment by deevus** [comment]
Ok should I re-create the issue there?

**Comment by pbogut** [comment]
Please.
